mixin/thanos-mixin/config.libsonnet

```diff
@@ -0,0 +1,41 @@
+{
+  _config+:: {
+    thanosQuerierJobPrefix: 'thanos-querier',
```

Comment:
`thanos-query` I think if we follow the naming in docs as well as other components here `receive/receiver` `compact/compactor`

Reply:
unless it's lots of work. It's a minor

Reply:
I can do it, I'll do that.